main.go

```diff
@@ -186,12 +200,14 @@ func toSpanner(driver, projectID, instanceID, dbName string, ioHelper *ioStreams
 	return nil
 }
 
-func schemaConv(driver string, ioHelper *ioStreams) (*internal.Conv, error) {
+func schemaConv(driver, tables string, sampleSize int64, ioHelper *ioStreams) (*internal.Conv, error) {
```

Comment:
Maybe drop sampleSize since it is only relevant for DynamoDB, and have schemaFromDynamoDB access the flag directly?

Reply:
Fixed.